src/Splunk.Metrics.Abstractions/NullStatsPublisher.cs

```diff
@@ -7,27 +8,27 @@ namespace Splunk.Metrics.Abstractions
     [SuppressMessage("ReSharper", "UnusedMember.Global")]
     public class NullStatsPublisher : IStatsPublisher
     {
-        public Task GaugeAsync(string bucket, double value) => Task.CompletedTask;
+        public Task GaugeAsync(string bucket, double value, IEnumerable<KeyValuePair<string,string>> additionalDimensions = null) => Task.CompletedTask;
```

Comment:
Ideally, IEnumerable should not be null, but can understand why this is done here.